common/setups/rasr/gmm_system.py

```diff
@@ -245,12 +245,14 @@ def monophone_training(
             )
 
         action_sequence = meta.align_and_accumulate_sequence(
-            align_iter, 1, mark_accumulate=False, mark_align=False
+            align_iter,
+            1,
+            mark_accumulate=kwargs.pop("mark_accumulate", False),
+            mark_align=kwargs.pop("mark_align", False),
```

Comment:
I prefer to have this explicitly as input arguments to the job and not hide it in the kwargs.